src/app/body/github-link/github-link.component.ts

```diff
@@ -56,10 +78,49 @@ export class GithubLinkComponent implements OnInit {
           this.dataFetched = true;
         });
       }
+      else if (this.searchType == 'username') {
+        if (this.linkType=='Private'){
+          this.httpService.getGithubPrivateRepos(this.bearerToken).pipe(map(data => {
+            const objData =data as GitOrgData[];
+            return objData;
+          })).subscribe(data => {
+            console.log(data);
```

Comment:
remove console.log and subscribe should follow the rule of next, error, complete